src/components/SubscriptionCard.js

```diff
@@ -366,38 +420,53 @@ class SubscriptionCard extends React.Component {
             <div className='description'>{description}</div>
           </div>
           <div className='footer'>
+
             <div className='subscription'>
-              <div className={`amount-frequency ${subscription.isPastDue ? 'past-due' : ''}`}>
-                <span className='amount'><Currency value={subscription.totalAmount} currency={subscription.currency} /></span>
-                <span className='currency-frequency'> {subscription.currency.toLowerCase()}/{frequency} </span> 
-              </div>
+              {this.state.visibleState === this.stateConstants.editAmount &&
+                <div className="editAmount">
+                  <div className="inputAmount">
+                    <span className="currency">$</span>
+                    <input type="number" step="0.01" min="1" name="amount" value={this.state.amountValue || subscription.totalAmount / 100} onChange={this.handleAmountChange}/>
```

Comment:
Could we change step to be in full dollar increments? Cents makes it hard to use.

Reply:
I reproduced what we have in the `createOrder` page, using the `Tier` component and the `InputField` (currency) component.
https://github.com/opencollective/frontend/blob/master/src/components/InputField.js#L311

Do we want to also modify that?

And also, do we want to use the `InputField` component here?

Reply:
Yes let's modify that too. I think fine to use InputField component. Is there a downside to it?